src/main/scala/encry/network/DeliveryManager.scala

```diff
@@ -222,38 +253,85 @@ class DeliveryManager extends Actor with StrictLogging {
   /**
     * If node is not synced, send request to random peer with non-younger history, overwise to all peers with
     * non-younger history
+    *
     * @param modifierTypeId
     * @param modifierIds
     */
   def requestDownload(modifierTypeId: ModifierTypeId, modifierIds: Seq[ModifierId]): Unit = {
-    if (settings.influxDB.isDefined)
-      context.actorSelection("/user/statsSender") ! SendDownloadRequest(modifierTypeId, modifierIds)
-    if (!isBlockChainSynced)
-      Random.shuffle(statusTracker.statuses.filter(_._2 != Younger)).headOption.foreach(pI =>
-        expect(pI._1, modifierTypeId, modifierIds)
-      )
+    influxRef.foreach(_ ! SendDownloadRequest(modifierTypeId, modifierIds))
+    if (!isBlockChainSynced) {
+      logger.info(s"Unsorted peers: ${
+        statusTracker.statuses
+          .map(x => x._1 -> (PeerPriorityStatus.toString(x._2._2), x._2._1)).mkString(",")
+      }")
+      val sortedPeers: Vector[(ConnectedPeer, (HistoryComparisonResult, PeerPriorityStatus))] = statusTracker.statuses
```

Comment:
make method, which will return seq of connected peer from which we should ask modifiers